### PR TITLE
Add missing action_limit settings and new policy_limit settings

### PR DESCRIPTION
**Pull request by michel-laterman**
action_limit was added as part of https://github.com/elastic/fleet-server/pull/2994 which i think made it into 8.12.0
policy_limit is a part of https://github.com/elastic/fleet-server/issues/3008 which is new

### CODE REVIEW

File: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc [hunk 1]
```diff
@@ -75,6 +75,19 @@ this setting may improve performance.
 `server.limits`::
 `policy_throttle`:::
 How often a new policy is rolled out to the agents.
+Deprecated: Please use policy_limits settings.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc:78]:
```suggestion
How often a new policy is rolled out to the agents.
+
Deprecated: Use the `policy_limit` settings instead.
```

File: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc [hunk 2]
```diff
@@ -75,6 +75,20 @@ this setting may improve performance.
 `server.limits`::
 `policy_throttle`:::
 How often a new policy is rolled out to the agents.
++
+Deprecated: Use the `policy_limit` settings instead.
+
+`action_limit.interval`:::
+How fast the fleet-server dispatches pending actions to the agents.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc:82]:
```suggestion
How quickly {fleet-server} dispatches pending actions to the agents.
```

File: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc [hunk 3]
```diff
@@ -75,6 +75,20 @@ this setting may improve performance.
...
 `policy_throttle`:::
 How often a new policy is rolled out to the agents.
++
+Deprecated: Use the `policy_limit` settings instead.
+
+`action_limit.interval`:::
+How quickly {fleet-server} dispatches pending actions to the agents.
+
+`action_limit.burst`:::
+Burst of actions that may be dispatched befoe falling back to the rate limit defined by `interval`.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc:85]:
```suggestion
Burst of actions that may be dispatched before falling back to the rate limit defined by `interval`.
```

File: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc [hunk 4]
```diff
@@ -75,6 +75,20 @@ this setting may improve performance.
...
+Deprecated: Use the `policy_limit` settings instead.
+
+`action_limit.interval`:::
+How quickly {fleet-server} dispatches pending actions to the agents.
+
+`action_limit.burst`:::
+Burst of actions that may be dispatched before falling back to the rate limit defined by `interval`.
+
+`policy_limit.interval`:::
+How fast the fleet-server dispatches pending policies to the agents.
```
Comment (kilfoyle) [inline docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc:88]:
```suggestion
How quickly {fleet-server} dispatches pending policies to the agents.
```

**Review by kilfoyle — APPROVED** [review]
LGTM! Thanks for adding these @michel-laterman. I'll merge this once the CI checks pass.

**Review by karenzone — APPROVED** [review]

**Pull request merged**